### PR TITLE
Fix #53: update webpack package version

### LINKED ISSUE #53: Webpack peer dependency needs to be updated
Thank you for your recent update, but when I tried to pull it with `npm`, I got this error:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: web@1.0.0
npm ERR! Found: webpack@5.31.0
npm ERR! node_modules/webpack
npm ERR!   dev webpack@"^5.27.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer webpack@"^4.0.0" from webpack-userscript@2.5.7
npm ERR! node_modules/webpack-userscript
npm ERR!   dev webpack-userscript@"^2.5.6" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

Checking the `package.json`, it seems like the peer dependency to an outdated webpack version causes trouble. Please consider to update your dependencies.

### PR DESCRIPTION
**Pull request by WangNianyi2001**
As mentioned in #53, current `webpack-userscript` is no longer compatible with `webpack@5.x.x`.
Therefore I committed an update to `webpack` & `webpack-cli` to the latest versions.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,11 +40,11 @@
...
     "tap-difflet": "^0.7.2",
     "tape": "^4.10.1",
-    "webpack": "^4.41.5",
-    "webpack-cli": "^3.3.10"
+    "webpack": "^5.74.0",
+    "webpack-cli": "^4.10.0"
   },
   "peerDependencies": {
-    "webpack": "^4.0.0"
+    "webpack": "^5.74.0"
```
Comment (Peeja) [inline package.json:47]: I think this would be `"4.x || 5.x"`, unless it's actually stopped working on Webpack 4.

**Comment by momocow** [comment]
Close without merge since this will be done in #79

**Comment by WangNianyi2001** [comment]
Glad to see this finally being resolved!

**Comment by momocow** [comment]
Thanks for your patience!

**Pull request closed without merging**